asset-registry/Cargo.toml

```diff
@@ -57,34 +57,68 @@ orml-xcm-support = { path = "../xcm-support", default-features = false }
 orml-xtokens = { path = "../xtokens" }
 
 [features]
-default = ["std"]
+default = [ "std" ]
 std = [
-	"serde",
-
-	"parity-scale-codec/std",
+	"cumulus-pallet-dmp-queue/std",
+	"cumulus-pallet-parachain-system/std",
+	"cumulus-pallet-xcm/std",
+	"cumulus-pallet-xcmp-queue/std",
+	"cumulus-primitives-core/std",
 	"frame-support/std",
 	"frame-system/std",
+	"log/std",
 	"orml-traits/std",
+	"orml-xcm-support/std",
+	"pallet-balances/std",
+	"pallet-message-queue/std",
 	"pallet-xcm/std",
+	"parachain-info/std",
+	"parity-scale-codec/std",
+	"polkadot-parachain-primitives/std",
+	"polkadot-runtime-common/std",
+	"polkadot-runtime-parachains/std",
 	"scale-info/std",
+	"serde",
+	"sp-core/std",
+	"sp-io/std",
 	"sp-runtime/std",
 	"sp-std/std",
 	"xcm-builder/std",
 	"xcm-executor/std",
 	"xcm/std",
 ]
 runtime-benchmarks = [
+	"cumulus-pallet-parachain-system/runtime-benchmarks",
+	"cumulus-pallet-xcmp-queue/runtime-benchmarks",
 	"frame-support/runtime-benchmarks",
 	"frame-system/runtime-benchmarks",
+	"orml-tokens/runtime-benchmarks",
+	"orml-xtokens/runtime-benchmarks",
+	"pallet-balances/runtime-benchmarks",
+	"pallet-message-queue/runtime-benchmarks",
 	"pallet-xcm/runtime-benchmarks",
+	"polkadot-parachain-primitives/runtime-benchmarks",
+	"polkadot-runtime-common/runtime-benchmarks",
+	"polkadot-runtime-parachains/runtime-benchmarks",
 	"sp-runtime/runtime-benchmarks",
 	"xcm-builder/runtime-benchmarks",
 	"xcm-executor/runtime-benchmarks",
-	"polkadot-runtime-parachains/runtime-benchmarks",
-	"polkadot-runtime-common/runtime-benchmarks",
 ]
 try-runtime = [
+	"cumulus-pallet-dmp-queue/try-runtime",
+	"cumulus-pallet-parachain-system/try-runtime",
+	"cumulus-pallet-xcm/try-runtime",
+	"cumulus-pallet-xcmp-queue/try-runtime",
 	"frame-support/try-runtime",
 	"frame-system/try-runtime",
+	"orml-tokens/try-runtime",
```

Comment:
this is causing issues when I try to publish this to crates.io using `cargo release`. orml-tokens is only a dev-dependencies so I think there is no reason to enable features for it?

Reply:
Hm, maybe it should be an optional feature enable then like `orml-tokens?/try-runtime`.  
I will try to debug it.

Reply:
No the optional is not possible to compile then. I think in the past, Parity stripped dev-dependencies before publishing from the `Cargo.toml` files. Not sure if this still happens.

Reply:
I tried and not working. I believe it is `cargo release` removes dev-dependencies before publish and all those features will not be valid.
Besides, we don't really need to enable them for dev-dependencies anyway.

Reply:
There is a first version of this here https://github.com/ggwpez/zepter/pull/56 for `zepter transpose features strip-dev-only`.  
I will write some more tests tomorrow and fix a few edge-cases but it seems to work when i tested it with `cargo publish --dry-run` on the crates here.